packages/venia-concept/src/.storybook/preview-body.html

```diff
@@ -1,103 +1,75 @@
 <!-- The contents of this file are copied from default-font-include.mst which is necessary to ensure the font matches as expected in storybook. -->
 <style type="text/css">
-    /* vietnamese */
-    @font-face {
-        font-family: 'Muli';
+    * cyrillic-ext */ @font-face {
+        font-family: 'Open Sans';
         font-style: normal;
         font-weight: 400;
         font-display: swap;
-        src: url(https://fonts.gstatic.com/s/muli/v20/7Aulp_0qiz-aVz7u3PJLcUMYOFnOkEk40e6fwniDtzNAAw.woff)
-            format('woff');
-        unicode-range: U+0102-0103, U+0110-0111, U+0128-0129, U+0168-0169,
-            U+01A0-01A1, U+01AF-01B0, U+1EA0-1EF9, U+20AB;
+        src: url(https://fonts.gstatic.com/s/opensans/v20/mem8YaGs126MiZpBA-UFWJ0bbck.woff2)
```

Comment:
Did you copy the content from the [link in the issue](https://fonts.googleapis.com/css?family=Open+Sans:400&display=swap)? I'm showing a diff locally between these lines in all the font-face blocks:

`src: url(https://fonts.gstatic.com/s/opensans/v20/mem8YaGs126MiZpBA-UFWJ0bf8pkAp6a.woff2) format('woff2');`